backend/src/main/java/de/neuefische/paulkreft/backend/exception/GlobalExceptionHandler.java

```diff
@@ -55,9 +55,15 @@ public String handleException(UsernameNotFoundException exception) {
         return exception.getMessage();
     }
 
-    @ExceptionHandler({LobbyNotFoundException.class})
+    @ExceptionHandler({LobbyGoneException.class})
+    @ResponseStatus(HttpStatus.GONE)
+    public String handleException(LobbyGoneException exception) {
+        return exception.getMessage();
+    }
+
+    @ExceptionHandler({RequestQueueNotFoundException.class})
     @ResponseStatus(HttpStatus.NOT_FOUND)
```

Comment:
Ich habe das Gefühl, dass NOT_FOUND nicht der beste Response Code für diesen Fall ist. Zumindest aus API Nutzersicht. Das NOT_FOUND signalisiert, dass die abgefragte Resource (z.B. eine Lobby) nicht existiert. Status Code 500 (oder ein anderer aus der 500er Reihe) passt meinem Verständnis nach besser. Die Queue ist interner Bestandteil deiner App und in dem Fall der Exception ist etwas damit nicht in Ordnung.

Reply:
Wie sicher bist du dir da? Sonst würde ich dazu auch nochmal Florian befragen. Hab damit nämlich auch gestruggled :)

Reply:
Sich bin ich mir nicht. Fühlt sich für mich sehr schlüssig an.